utils/wallet.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+class Wallet:
+    def __init__(self, user_id, user_wallet, user_balance):
+        """
+        Initialize a Wallet object.
+
+        :param id: Unique identifier for the Wallet object
+        :type id: int
+        :param user_id: Unique identifier for the user the wallet belongs to
+        :type user_id: int
+        :param user_wallet: Wallet address of the user
+        :type user_wallet: str
+        :param user_balance: Balance of the user in the wallet
+        :type user_balance: float
+        """
+        self.id = self.generate_wallet_address()
+        self.user_id = user_id
+        self.user_wallet = user_wallet
+        self.user_balance = user_balance
+
+    def __repr__(self):
+        
+        """
+        Return a string representation of the Wallet object.
+
+        The string representation should be formatted like this:
+        Wallet(id=1, user_id=123, user_wallet='WalletABC', user_balance=100.0)
+
+        :return: A string representation of the Wallet object
+        :rtype: str
+        """
+        return (f"Wallet(id={self.id}, user_id={self.user_id}, "
+                f"user_wallet='{self.user_wallet}', user_balance={self.user_balance})")
+    
+    def generate_wallet_address(self):
```

Comment:
It might be worth revisting the generate wallet method, instead of randomly generating a 13 digit number, we might want to generate something more specific to the user. For example if my number is 123456789, the wallet address can be "rafiki.last_4_digits.NAME" or something along those lines